src/pages/gen2/build-a-backend/data/custom-business-logic/index.mdx

```diff
@@ -169,72 +167,192 @@ export const data = defineData({
     defaultAuthorizationMode: 'apiKey',
     apiKeyAuthorizationMode: {
       expiresInDays: 30
-    }
+    },
   },
   functions: {
-    // Step 3 - Map the function key to the function definition
-    echoHandler: defineFunction({
-      entry: './echo-handler.ts'
-    })
+    'echoHandler': echoHandler
   }
 });
 ```
 
-## Step 3 - Invoke the custom query or mutation
+</Block>
+<Block name="Custom resolver powered by AppSync JavaScript resolvers">
 
-<Callout info>
+Custom resolvers work on a "request/response" basis, you choose a data source, map your request to the data sources' input parameters, and then map the data source's response back to the query/mutation's return type. Custom resolvers provide the benefit of no cold starts, less infrastructure to manage, and no additional charge for Lambda function's execution. Review [Choosing between custom resolver and function](https://docs.aws.amazon.com/appsync/latest/devguide/resolver-reference-overview-js.html#choosing-data-source).
```

Comment:
>  and no additional charge for Lambda function's execution

is this referring to a single function "for a Lambda function's execution"
or multiple "for Lambda functions' execution"